roles/devscripts/tasks/310_prepare_overlay.yml

```diff
@@ -21,6 +21,18 @@
     uri: "qemu:///system"
   register: _current_vms
 
+- name: Initiate graceful shutdown of the cluster.
+  vars:
+    cifmw_openshift_adm_op: "shutdown"
+    cifmw_openshift_adm_cert_expire_date_file: >-
+      {{
+        (
+          cifmw_devscripts_config.working_dir, '.ocp_cert_not_after'
+        ) | ansible.builtin.path_join
+      }}
+  ansible.builtin.import_role:
+    name: openshift_adm
```

Comment:
I think this one actually shuts the VM down on its own? Meaning the next task may be useless (`virsh destroy` kind of pull the power cord out)?

Reply:
sounds good! Drained SSH doesn't mean "host down" indeed.